features/org.eclipse.pde-feature/p2.inf

```diff
@@ -0,0 +1,10 @@
+# Require the annotation bundles so they get included when installing PDE
+requires.0.namespace = osgi.bundle
+requires.0.name = org.osgi.annotation.versioning
+requires.0.greedy = true
+requires.0.optional = true
+
+requires.1.namespace = osgi.bundle
+requires.1.name = org.osgi.annotation.bundle
+requires.1.greedy = true
+requires.1.optional = true
```

Comment:
why not simply adding them into the feature.xml?

Reply:
We only want to include them if they are present and do not require them in general.

Reply:
I don't think it's worth the complexity. Those would better be always part of the feature IMO. It makes everything simpler for maintenance and usage, and adds a very small footprint.

Reply:
As long as we finally get those in I'm fine with it, can you provide a PR with adding them to the features? Last time i tried, they where not available in the SDK even though the are added to the prereq target... Or do we need to add/commit this and then wait for the next I-Build?

Reply:
The .target is deployed with https://ci.eclipse.org/releng/job/deploy-eclipse-sdk-target-pom/ ; so if the change is in an this build has run, then the bundes should be in and available for inclusion in features.

Reply:
It was added with this commit: https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/pull/420

Reply:
OK, so now it should be in, while it may not have been the case in the few minutes after .target change was merged.

Reply:
I did update my SDk to latest version and performed the setup task but still PDE does not offer me these bundles, I'll try to ad them by hand and the lets see if jenkins complains...

Reply:
I have them when looking at the "Content" tab of the eclipse-sdk-prereqs.target.